test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/accounts/homepage/HomePageControllerSpec.scala

```diff
@@ -215,6 +215,290 @@ class HomePageControllerSpec
 
       def extractAmount(s: String): String = s.substring(s.indexOf('£'))
 
+      // the callToActionButton selector is designed to select the only
+      // button on the page for us to check which message it has
+      val callToActionButton    = "#content a.button"
+      val resumeDraftMessage    = "drafts.list.resume"
+      val viewAndPayMessage     = "returns.list.viewAndPay"
+      val startNewReturnMessage = "account.home.button.start-a-new-return"
+
+      "display the resume draft link as a button when there is a draft and no sent returns" in {
+        val sampleDraftReturn = sample[DraftSingleDisposalReturn]
+        val subscribed        =
+          sample[Subscribed].copy(
+            draftReturns = List(sampleDraftReturn),
+            sentReturns = List.empty[ReturnSummary]
+          )
+
+        inSequence {
+          mockAuthWithNoRetrievals()
+          mockGetSession(
+            SessionData.empty.copy(
+              userType = Some(UserType.Individual),
+              journeyStatus = Some(subscribed)
+            )
+          )
+        }
+
+        checkPageIsDisplayed(
+          performAction(),
+          messageFromMessageKey("account.home.title"),
+          doc =>
+            doc
+              .select(callToActionButton)
+              .text should include(
```

Comment:
`shouldBe(...)` is a stronger check than `should include(...)`

Reply:
yes agreed, done